api/catalog/api/utils/validate_images.py

```diff
@@ -145,6 +151,6 @@ def validate_images(query_hash, start_slice, results, image_urls):
     )
 
 
-def _validation_failure(request, exception):
+def _validation_failure(exception):
```

Comment:
Nit: can we name this function with a verb, something like `_log_validation_failure`?